module-audio/Audio/Profiles/README.md

```diff
@@ -30,13 +32,13 @@ appropriate filter type - `None`.
 
 ### Array element structure
 
-| Field      | Value type | Description                                                |
-|------------|------------|------------------------------------------------------------|
-| filterType | string     | Defines the type of the filter                             |
-| frequency  | float      | Defines the cutoff frequency for the filter                |
-| samplerate | integer    | Defines the samplerate used to calculate the filter coeffs |
-| Q          | float      | Defines Q-factor for the filter                            |
-| gain       | float      | Defines gain for shelf-like filters                        |
+| Field      | Value type | Description                                                | Valid range    |
+|------------|------------|------------------------------------------------------------|----------------|
+| filterType | string     | Defines the type of the filter                             | See list below |
+| frequency  | float      | Defines the cutoff frequency for the filter                | \> 0Hz         | 
+| samplerate | integer    | Defines the samplerate used to calculate the filter coeffs | \> 0Hz         |
+| Q          | float      | Defines Q factor for the filter                            | 0.1 - 10       |
+| gain       | float      | Defines gain for shelf-like filters                        | \> 0           |
```

Comment:
```suggestion
| frequency  | float      | Defines the cutoff frequency for the filter                | ≥ 0Hz         | 
| samplerate | integer    | Defines the samplerate used to calculate the filter coeffs |≥ 0Hz         |
| Q          | float      | Defines Q factor for the filter                            | 0.1 - 10       |
| gain       | float      | Defines gain for shelf-like filters                        | ≥ 0           |
```

Reply:
I've checked it on Github and it renders properly, so IMO it can be left as it is.